packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/index.md

```diff
@@ -17,6 +17,11 @@ Enable a mobile app to verify a user identity for an Okta custom authenticator.
 * An Okta developer app
 * <StackSnippet snippet="notifservicelink" inline />
 * <StackSnippet snippet="appreq" inline />
+* (Optional) Access to the following APIs:
```

Comment:
Need to add link to https://developer.okta.com/docs/reference/api/authorization-servers/#update-a-policy-rule for updating Authorization Server policy rule grant types

Reply:
Done. Thanks!